spec/io/io_spec.rb

```diff
@@ -34,6 +34,16 @@
       end
     end
 
+    context ".from_csv_url" do
+      it "loads from a CSV file" do
+        df = Daru::DataFrame.from_csv_url('https://raw.githubusercontent.com/anshuman23/ruBayes/master/test/sample.csv')
```

Comment:
Better use some mock library (webmock or VCR) to test network interactions. Imagine somebody developing new features and restarting specs every now and then -- and each time waiting for this small spec to connect to network and download test data (which also could change or vanish in future).